base/unicode/utf32.jl

```diff
@@ -219,9 +219,9 @@ isvalid(str::Vector{Char}) = isvalid(UTF32String, str)
 
 utf32(x) = convert(UTF32String, x)
 
-utf32(p::Ptr{Char}, len::Integer) = utf32(pointer_to_array(p, len))
-utf32(p::Union{Ptr{UInt32}, Ptr{Int32}}, len::Integer) = utf32(convert(Ptr{Char}, p), len)
-function utf32(p::Union{Ptr{Char}, Ptr{UInt32}, Ptr{Int32}})
+utf32(p::Ptr{UInt32}, len::Integer) = utf32(pointer_to_array(p, len))
+utf32(p::Union{Ptr{Char}, Ptr{Int32}}, len::Integer) = utf32(convert(Ptr{UInt32}, p), len)
+function utf32(p::Union{Ptr{UInt32}, Ptr{Char}, Ptr{Int32}})
```

Comment:
I like how `UInt32` asserts dominance here by being first in the `Union`. :rage2: